docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -4183,6 +4183,28 @@ Apply the proper user rights and restore your database from your backup
 
 Start packetfence and make sure your stats are still there and being collected properly. Then remove the backup you made `rm -fr /usr/local/pf/var/graphite.bak/`.
 
+Troubleshooting
+~~~~~~~~~~~~~~~
+
+This section will adress specific problems and known solutions.
+
+"Internet Explorer cannot display the webpage"
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Problem: Internet Explorer 8-10 will probably raise "Internet Explorer cannot display the webpage" errors when attempting to access PacketFence administration interface. 
+
+Troubleshoot: 
+
+ * Verify that PacketFence administration interface is started:
+
+  # cd /usr/local/pf
+  # bin/pfcmd service httpd.admin start
+ 
+Solution: 
+ 
+* TLSv1.2 needs to be activated for PacketFence admininstation interface to be accessible. It is advised that you update your browser. If not possible: 
+
+  Within  Internet Explorer: click Tools, Internet Options, Advanced and make sure that TLS v1.2 is enabled under the security section. Retry.
+
```

Comment:
This would be better worded as two separate issues.

* PacketFence administration interface is not started:
  - How to fix it
* TLSv1.2 in Internet Explorer needs to be activated.
  - How to fix it